### ISSUE TITLE
[CAL-3914] Patch request to confirm a booking doesn't run the business logic

### ISSUE THREAD

**Issue by martinfeuillet** [calcom/cal.com#15389]
### Is your proposal related to a problem?

When we do a patch request to confirm a pending booking, all the business logic aren't triggered.
I explain : 
There is a problem when the booking needs to be confirmed.
When it's this scenario, i do a patch on the booking by doing :
{{baseUrl}}/bookings/:id and changing the status from pending to accepted, a link is not generated, and i also don't have any mail that are generated to warn the user and the owner of the booking that this one is accepted.

### Describe the solution you'd like

I would like to have a google meet link and mails sent to the user and the owner of the meeting, exactly as when we book a simple booking without any confirmation

### Describe alternatives you've considered

(Write your answer here.)

### Additional context

(Write your answer here.)

### Requirement/Document

(Share it here.)

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

<sub>[CAL-3914](https://linear.app/calcom/issue/CAL-3914/patch-request-to-confirm-a-booking-doesnt-run-the-business-logic)</sub>

**Comment by GeorgeShao** [comment]
I ran into this earlier today as well, not sure if it's a bug or expected functionality. Either way, I'd prefer that there be an option whether to run the business logic (creating meeting link, creating calendar event, sending confirmation email, etc) or not. 

In general, it would be nice to be able to enable/disable all calendar event creation & meeting link creation, for more control.

**Comment by Shaik-Sirajuddin** [comment]
It would be nice to have separate endpoints to **confirm , reschedule booking** that trigger business logic 
@PeerRich Could you confirm separate endpoints should be built for this

**Comment by martinfeuillet** [comment]
Any news ?

**Comment by martinfeuillet** [comment]
Any news ?

**Comment by CarinaWolli** [comment]
Not sure if that's still relevant

@JDP-Neo Maybe you can take a look, if not relevant feel free to close 🙏
